frontend-react/src/components/landingpage.js

```diff
@@ -1,12 +1,58 @@
 import React, { Component } from 'react';
+import Navigation from '../navigation.js';
 import '../style/landingpage.css';
 
 class LandingPage extends Component {
+    handleClick() {
+        console.log('Click happened');
+    }
     render() {
         return (
-            <div><h1 className="code">Landing Page</h1></div>
+            
+            <div style={{display: "flex"}}>
+                <div style={{width: "50%"}}>
+                    <img src={require('../style/link2.png')}></img>
+                </div>
+                <form style={{width: "50%", padding: "1rem"}} >
+                    <h1 style={{color: "white"}}>User Friendly and nbsp</h1>
+                    <h2 style={{color: "white"}}>Deep Learning</h2>
+
+                    <div style={styles.rectangle}>
+                        <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginLeft: "1rem"}, {marginBottom: "2rem"}} for= {'username'}>Username</label>
+                            <input id={'username'}></input>
+                        </div>
+
+                         <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "0.1rem"}} for={'password'}>Password</label>
+                            <input id={'password'} type={'password'}></input>
+                        </div>
+
+                        <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "2.5rem"}} for={'port'}>Port</label>
```

Comment:
Styling should be applied in style/landingpage.css
The source images should also be given a better name and be put in a new folder such as "resources" or something like that.